packages/wp-dashboard/src/api/story.js

```diff
@@ -160,6 +160,15 @@ export const createStoryFromTemplate = async (config, template) => {
       status: 'auto-draft',
       featuredMedia: {
         id: 0,
+        url: '',
+      },
+      title: '',
+      publisherLogo: {
+        url: '',
+      },
+      pages: [],
+      metadata: {
+        publisher: '',
```

Comment:
Fixes proptype error when selecting a template.

https://github.com/GoogleForCreators/web-stories-wp/blob/2a2e69f9cc532d444a8f9ac90e53fd46ac815d89/packages/story-editor/src/output/story.js#L100-L121

Reply:
Did this error occur when running e2e tests?

> This error was visible locally but not in CI.

FYI, prop types errors are only displayed when building in development mode. E2e tests are run in production mode.

Reply:
Oh! Okay I understand now, thanks!